lib/go-tc/users.go

```diff
@@ -253,14 +253,14 @@ type UserV40 struct {
 	Username string `json:"username" db:"username"`
 }
 
-// UsersResponseV4 is the type of a response from Traffic Ops to requests made
+// UsersResponseV4 is the type of response from Traffic Ops to requests made
 // to /users which return more than one user for the latest 4.x api version variant.
 type UsersResponseV4 struct {
 	Response []UserV4 `json:"response"`
 	Alerts
 }
 
-// UserResponseV4 is the type of a response from Traffic Ops to requests made
+// UserResponseV4 is the type of response from Traffic Ops to requests made
```

Comment:
These two changes aren't any more or less grammatically correct than the original, but I think the meaning of the word `type` in this context is clearer before the change.